.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Lint with pylint
+        run: |
+          pylint -E --rcfile=./.pylintrc app/ main.py
+      - name: Test with pytest
+        run: |
+          pytest
```

Comment:
https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e
https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

Reply:
please configure your IDE, so you always include the EOL